### ISSUE TITLE
Example on front page hard to follow

### ISSUE THREAD

**Issue by dstansby** [spacetelescope/gwcs#363]
I am coming to `gwcs` as a first time user, and thought the first two narrative sections on the documentation page (https://gwcs.readthedocs.io/en/latest) were a great introduction to `gwcs`.

I found the example in the third section ([A step by step example of constructing an imaging GWCS object.](https://gwcs.readthedocs.io/en/latest/#a-step-by-step-example-of-constructing-an-imaging-gwcs-object)) hard to follow though, because it is focused on precisely replicating a `fitswcs`, instead of explaining the concepts and API of constructing a `gwcs`.

My suggestion for improvement would be to move this example out of the introductory section, and replace it with a simple example that uses only one transformation step, but still shows how the `gwcs` API works in practice.

**Comment by perrygreenfield** [comment]
Addressed by #483
